packages/oas-to-har/src/index.js

```diff
@@ -81,10 +81,10 @@ module.exports = (
     httpVersion: 'HTTP/1.1',
   };
 
-  const proxyEnabled = extensions.getExtension(extensions.PROXY_ENABLED, oas, operation);
-  // TODO look to move this to Oas class as well
-  if (proxyEnabled && opts.proxyUrl) {
-    har.url = `https://try.readme.io/${har.url}`;
+  if (opts.proxyUrl) {
```

Comment:
Do we not need to check for `proxyEnabled` any longer?

Reply:
`proxyEnabled` was the result of the `getExtension` call and is still in use on line 85.